wagtail/contrib/modeladmin/tests/test_simple_modeladmin.py

```diff
@@ -263,6 +263,43 @@ def test_col_extra_attributes(self):
         """
         self.assertContains(response, test_html, html=True)
 
+    def test_linked_name_column(self):
+        response = self.get()
+        self.assertEqual(response.status_code, 200)
+        # The InspectView is enabled in the AuthorModelAdmin, so the inspect-button is
+        # used for the link.
+        test_html = """
+            <div class="title-wrapper"><a href="/admin/modeladmintest/author/inspect/1/" title="Inspect this author">J. R. R. Tolkien</a></div>
+        """
+        self.assertContains(response, test_html, html=True)
+
+    @mock.patch(
+        "wagtail.contrib.modeladmin.helpers.permission.PermissionHelper.user_can_inspect_obj",
+        return_value=False,
+    )
+    def test_linked_name_column_without_inspect_view(self, *mocks):
```

Comment:
```suggestion
    def test_title_column_links_to_inspect_view_when_user_cannot_edit(self, *mocks):
```

Reply:
I will commit your suggestion with GH, but will properly change the tests locally (probably on monday).

Reply:
Absolutely. Whenever you have the time is good by me @th3hamm0r.